### DISCUSSION TITLE
如何在一句话中got两个参数

### DISCUSSION THREAD

**Discussion by A1in-SY** [nonebot/discussions#227] [问答] [answered]
比如我发送 test 1 2
能把test识别为指令，1为一个参数，2为另一个参数
现在是这样子写的
plain_text = args.extract_plain_text()
if " " in plain_text:
    matcher.set_arg("arg1", Message(plain_text.split[" "][0]))
    matcher.set_arg("up", Message(plain_text.split[" "][1]))
但是会报NotImplementedError

**Comment by yanyongyu** [comment, marked as answer]
需要你自己parse命令，就像你现在写的那样。或者使用on_shell_command，不过没啥区别。
